### PR TITLE
Sentry: remove DSN value from codebase

### PR DESCRIPTION
**Pull request by jpcloureiro**
**Description**

Remove Sentry DSN from the codebase & load it from environment variables.

Sentry SDK won't send any events when DSN is not defined

For non Metamask developers there's a reference in the README markdown on what to do if they want to log app events on their own Sentry project.

**Issue**

Progresses https://github.com/metamask/mobile-planning/issues/580

### CODE REVIEW

**Comment by jpcloureiro** [comment]
@MetaMask/mobile-devs do we need to update `bitrise.yml` to get the new env var?

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ cd metamask-mobile
   cp .js.env.example .js.env
 ```
 -   _Non-MetaMask Only:_ Create an account and generate your own API key at [Infura](https://infura.io) in order to connect to main and test nets. Fill `MM_INFURA_PROJECT_ID` in `.js.env`. (App will run without it, but will not be able to connect to actual network.)
+- _Non-MetaMask Only:_ Fill `MM_SENTRY_DSN` in `.js.env` if you want the app to emit logs to your own Sentry project.
```
Comment (sethkfman) [inline README.md:96]: @jpcloureiro If this value doesn't exist does the app crash?

Comment (jpcloureiro) [reply README.md:96]: Sentry SDK can handle empty DSN values. 

https://docs.sentry.io/product/sentry-basics/dsn-explainer/#what-the-dsn-does

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**